lib/std/jsbigints.nim

```diff
@@ -0,0 +1,207 @@
+## Arbitrary precision integers.
+## * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
+when not defined(js) and not defined(nimdoc):
+  {.fatal: "Module jsbigints is designed to be used with the JavaScript backend.".}
+
+type JsBigInt* = ref object of JsRoot ## Arbitrary precision integer for JavaScript target.
+
+func big*(integer: SomeInteger): JsBigInt {.importjs: "BigInt(#)".} =
```

Comment:
"big"`? We need a better name.

Reply:
hahahahaha, I named it `newBigInt` originally, but then we renamed to `big`, but yeah suggestions welcome :thinking:

Reply:
@Araq No better naming found, then I used `newJsBigInt`, later we can add a short alias anyway.